### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by emmatrain**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+
+.diagonal-box-one {
+	background-image: linear-gradient(45deg, #ed4264, #ffedbc); /*dark pink into yellow*/	
+	transform: skewY(-11deg);
+	padding: 100px;
+}
+
+.content {
+	max-width: 100em;
```
Comment (charliecog) [inline styles.css:24]: not sure this is doing anything, worth a check and potential delete

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+	margin: 0 auto;
```
Comment (charliecog) [inline styles.css:25]: same as line above

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+	right: 50px;
+	top: 90%;
+}
+
+/* NAV STYLING */
+
+nav {
+	margin: 50px 150px 10px;
+	display: flex;
+	justify-content: flex-end;
```
Comment (charliecog) [inline styles.css:41]: potential for deletion

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+nav a, footer a:visited {color:#ed4264;} /*dark pink*/
+nav a, footer a:hover   {color:#ed4264;}
+nav a, footer a:active  {color:#ed4264;} /*dark pink*/
+
+
+/* HEADER STYLING */
+
+header {
+	margin: 120px 100px;
+	background-color: transparent;
```
Comment (charliecog) [inline styles.css:60]: doesnt appear to do anything, remove

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+header img {
+	height: 32px;
+	width: 32px;
+	margin: 10px;
+}
+
+.social-icons {
+	display: flex;
+	justify-content: flex-start;
```
Comment (charliecog) [inline styles.css:109]: this is the default value so doesnt need to be defined, remove

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+	justify-items: center;
+}
+}
+
+
+@media only screen and (max-width: 650px) {
+.responsive-nav {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
```
Comment (charliecog) [inline styles.css:345]: doesnt appear to do anything, remove

File: index.html [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+			<section class=responsive-nav>
+				<li><a href="index.html">Home</a></li>
+				<li><a href="#about">About</a></li>
+				<li><a href="#projects">Projects</a></li>
+				<li><a href="#contact">Get in touch</a></li>
+			</section>
+
+		<!-- HEADER CONTENT (RESPONSIVE) -->
+
+			<section class=responsive-intro>
```
Comment (charliecog) [inline index.html:72]: This responsive-intro content feels repeated from what was above - things that can be reused should be, have a look and see if you can use what you already have and hide/show/manipulate using css

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,373 @@
...
+	}
+	
+	.headshot {
+		height: 250px;
+		width: 250px;
+	}
+		
+	#about {
+		padding: 30px 0 0 0;
+		margin: 10px 100px 100px;
```
Comment (charliecog) [inline styles.css:276]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
+		font-size: 1.5rem;
+	}
+	
+	.headshot {
+		height: 250px;
+		width: 250px;
+	}
+		
+	#about {
+		padding: 30px 0 0 0;
```
Comment (charliecog) [inline styles.css:275]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,373 @@
...
+		margin: 10px 100px 100px;
+		display: block;
```
Comment (charliecog) [inline styles.css:277]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,373 @@
...
+		font-family: 'Montserrat', serif;
```
Comment (charliecog) [inline styles.css:278]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,373 @@
...
+		font-size: 1rem;
```
Comment (charliecog) [inline styles.css:279]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,373 @@
...
+		justify-items: center;
```
Comment (charliecog) [inline styles.css:280]: this is already defined in the 1160px media query - can be removed

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,373 @@
...
+.bio-content p {
+	font-family: 'Montserrat', sans-serif;
+	font-size: 1.5rem;
+	width: 700px;
+}
+
+/*PROJECTS STYLING*/
+
+#projects {
+	width: 100%;
```
Comment (charliecog) [inline styles.css:164]: section elements are block by default so dont require this declaration, can be removed

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,373 @@
...
+}
+
+#projects h2 span {
+	font-family: 'Londrina Outline';
```
Comment (charliecog) [inline styles.css:168]: Good practice is to always offer a fallback font as a comma separated list. Something websafe that is guaranteed to work.

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,373 @@
...
+	width: 200px;
+}
+
+.projects-text p {
+	font-family: 'Montserrat', sans-serif;
+	font-size: 1.5rem;
+	width: 700px;
+}
+
+.projects-container {
```
Comment (charliecog) [inline styles.css:187]: For this css selector if you make it:
```
.projects-container {
display: flex;
flex-flow: row wrap;
text-align: center;
}
```

then it should center

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,373 @@
...
+.hidden-repsonsive-sections {
+	display: none;
+}
+
+/*FOOTER STYLING*/
+
+footer {
+	background-color: #ffedbc;
+	height: 120px;
+	width: 100%;
```
Comment (charliecog) [inline styles.css:225]: dont need this one, can remove

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,373 @@
...
+		margin: 0;
+	}
+	
+	.responsive-intro h1 {
+		font-size: 3rem;
+		color: #ed4264;
+	}
+	
+	.responsive-intro h1 span {
+			font-family: 'Londrina Outline';
```
Comment (charliecog) [inline styles.css:256]: formatting, untab this to bring it back in line, also worth adding a fallback font

**Review by charliecog — COMMENTED** [review]
Looking good, few tweaks requested - give me a shout if need any discussion on any of these points. Almost there!

General feedback:
- At 630px screen width all the navbars disappear

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,373 @@
...
+	border-radius: 50%;
+	height: 350px;
+	width: 330px;
+	object-fit: cover;
+}
+
+.bio-content p {
+	font-family: 'Montserrat', sans-serif;
+	font-size: 1.5rem;
+	width: 700px;
```
[the remark this answers is not in the document]
Comment (emmatrain) [reply styles.css:158]: This mucked up styling on desktop viewport, so added a media query at max 1380px to change it into a % and that seems to work...

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,373 @@
+/*GENERAL STYLING*/
+
+* {
+   box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%; /*16px (so 1 rem = 10px)*/
```
[the remark this answers is not in the document]
Comment (emmatrain) [reply styles.css:8]: Ok I read somewhere that % was better. No idea when or what it was now!!

File: styles.css (hunk 1 above)
Comment (emmatrain) [reply styles.css:24]: on my screen it aligns everything over to the right. If I delete the header text is way over to the left.

File: styles.css (hunk 2 above)
Comment (emmatrain) [reply styles.css:25]: same as above, if i delete it the same thing happens.

File: styles.css (hunk 3 above)
Comment (emmatrain) [reply styles.css:41]: aligns nav elements to the right on my view

**Comment by emmatrain** [comment]
> Looking good, few tweaks requested - give me a shout if need any discussion on any of these points. Almost there!
> 
> General feedback:
> 
> * At 630px screen width all the navbars disappear

I moved both the show/hide to 650.

**Review by charliecog — APPROVED** [review]
Great work!

**Pull request merged**